src/components/balance/modals/ModalSelectAccount.vue

```diff
@@ -21,7 +21,7 @@
           </div>
 
           <input
-            v-model="selAddress"
+            v-model="valueAddressOrWallet"
```

Comment:
![bandicam 2021-11-14 13-28-27-315](https://user-images.githubusercontent.com/40356749/141681234-f6db07bc-2e22-4770-be4f-78f8c873912f.jpg)

How about changing the icon in https://github.com/PlasmNetwork/astar-apps/blob/01831da90c52ff0f6db15d58461679c73c2ef820/src/components/balance/modals/ModalSelectAccount.vue#L18 based on the input account type? If the user inputs an H160, we display Ethereum icon, if not, we display the Polkadot identicon.